react/features/base/ui/components/web/Button.tsx

```diff
@@ -206,6 +206,7 @@ const Button = ({
             disabled = { disabled }
             { ...(id ? { id } : {}) }
             onClick = { onClick }
+            ref = { ref }
```

Comment:
When refactoring `LobbyParticipantItem` this error came to light. There it's used with a ref, but the Button didn't support it, but because one file was TS and one JS no error was shown

Reply:
👍